### DISCUSSION TITLE
My host server doesn't load the .GLB even with SSL

### DISCUSSION THREAD

**Discussion by renanmalato** [google/model-viewer#4683] [Q&A]
My website is hosted in Metaweb - it's a brazilian company

No matter where or what folder i upload the **.glb file it doesn't loads on model-viewer**

I have to upload in a external cloud and call it on my website

I heard about the SSL, so I paid the certified but nothing changed

Anyone have this same problem? Some trials for me?
Some kind of port issue?

-

The problem is there is no clouds that gets you a direct link anymore. I found one called Filebin but it keeps the file for 6 days only.

I appreciate any help. Thanks

**Comment by elalish** [comment]
Do you get a CORS error in your JS console? https://modelviewer.dev/docs/faq.html#entrydocs-technical-questions-cors
